cmd/collector/main.go

```diff
@@ -23,14 +23,13 @@ func main() {
 		log.Fatalf("failed to build default components: %v", err)
 	}
 
-	info := component.ApplicationStartInfo{
-		ExeName:  "collector",
-		LongName: "Hypertrace Collector",
-		Version:  Version,
-		GitHash:  GitHash,
+	info := component.BuildInfo{
+		Command:     "hypertrace-collector",
+		Description: "Hypertrace Collector",
+		Version:     Version,
```

Comment:
curious why the GitHash was removed.

Reply:
It is not supported anymore. We could check if version == dev then use dev-{hash}. Does that make sense?